examples/agent_motion_prediction/agent_motion_config.yaml

```diff
@@ -43,6 +43,11 @@ raster_params:
   # whether to completely disable traffic light faces in the semantic rasterizer
   disable_traffic_light_faces: False
 
+  # when set to True performs a flip inside the rasteriser.
```

Comment:
```suggestion
  # When set to True, the rasterizer will set the raster origin at bottom left, i.e. vehicles are driving on the right side of the road. With this change, the vertical flipping on the raster used in the visualization code is no longer needed.
  # Note that this will flip the model input raster and break previously trained models. 
  # If users want to restore the old behavior, please set the "set_origin_to_bottom" to False. Additional, users will also need to restore the vertical flipping on the raster in the visualisation code.
```